test/fluree/db/query/sparql_test.cljc

```diff
@@ -296,6 +311,24 @@
 ;; TODO
 #_(deftest parse-functions)
 
+(deftest subject-iri
+  (let [query
+        "PREFIX psm: <http://srv.ktbl.de/data/psm/>
+PREFIX ex: <http://example.org/>
+
+SELECT ?p ?o
+FROM <cookbook/base>
+WHERE {
+    ex:andrew ?p ?o. # Should provide just the triples related to andrew
```

Comment:
This comment might be more useful in a `testing` annotation?

Reply:
This was a direct test case from the ticket, I think I'll add it in with the `simple-triple` expectations.